x-pack/plugins/uptime/common/utils/es_search.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { estypes } from '@elastic/elasticsearch';
+
+export function createEsQuery<T extends estypes.SearchRequest>(params: T): T {
```

Comment:
I'm not understanding why we need this. Is it to save implementers from having to import `SearchRequest` in their files? We aren't using the generic type utility of this function in either of the callers, and are simply returning the provided params. Am I missing the purpose entirely?

Reply:
basically to provide intellicense for types params.